exclusion-patterns.txt

```diff
@@ -0,0 +1 @@
+# Put any regular expressions to exclude messages here, one regex per line
```

Comment:
will be nice to have a few commented out examples of trivial and less trivial cases

Reply:
Added a few basic regex's to the examples, with a link to a more in-depth tutorial on regular expressions